notebooks/3_KernelMethods.ipynb

```diff
@@ -510,7 +536,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Our regularization is {{regularization*lambda_max}}."
+    "Our regularization is `regularization * lambda_max`."
```

Comment:
The previously formatting should output the variable value. Why did you implement this change?

Reply:
Because it doesn't work on my Jupyter notebook. Do I need to install any plug-ins?

![image](https://user-images.githubusercontent.com/56018055/97764380-bd0a6800-1b0e-11eb-8c17-95d784ff7b46.png)

Reply:
This page should help: https://stackoverflow.com/questions/52812231/print-variable-in-jupyter-notebook-markdown-cell-python

Can you make a note in the notebook front-matter?

Reply:
Thank you! Now it works. 
![image](https://user-images.githubusercontent.com/56018055/97808063-bee63f80-1c64-11eb-9306-61e2b14c6a90.png)


I don't quite understand "make a note in the notebook front-matter". What exactly should I do?

Reply:
That means in the areas where we indicate what the dependencies are, we should include the jupyter contrib nbextensions.